### PR TITLE
Update EE 11 Liberty features for recent changes

### PR DESCRIPTION
**Pull request by jhanders34**
- Add beanValidation-3.1 and interceptor-2.2 features since they were added to the list of EE 11 features
- Remove managedBeans-2.0 feature from EE 11 features since the plan is to remove the feature from the Jakarta platform in EE 11.

### CODE REVIEW

**Comment by jhanders34** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/enterpriseBeansLite-4.0/io.openliberty.enterpriseBeansLite-4.0.feature [hunk 1]
```diff
@@ -12,7 +12,7 @@ Subsystem-Category: JakartaEE9Application
 -features=com.ibm.websphere.appserver.eeCompatible-9.0; ibm.tolerates:="10.0, 11.0", \
   com.ibm.websphere.appserver.contextService-1.0, \
   io.openliberty.ejbLiteCore-2.0, \
-  io.openliberty.jakarta.interceptor-2.0; ibm.tolerates:="2.1", \
+  io.openliberty.jakarta.interceptor-2.0; ibm.tolerates:="2.1 2.2", \
```
Comment (njr-11) [inline dev/com.ibm.websphere.appserver.features/visibility/public/enterpriseBeansLite-4.0/io.openliberty.enterpriseBeansLite-4.0.feature]: Does this work without the comma delimiter?  I haven't seen it done this way before so I'm wondering if it's a typo.

Comment (jhanders34) [reply dev/com.ibm.websphere.appserver.features/visibility/public/enterpriseBeansLite-4.0/io.openliberty.enterpriseBeansLite-4.0.feature]: No.  That was bad on my part.  I have corrected it.

**Comment by jhanders34** [comment]
#libby

**Review by njr-11 — DISMISSED** [review]
The feature changes look fine.  I'm not qualified to review the NLS properties files in the various languages.  I can point out that the english-only words Jakarta Bean Validation aren't fully present in 2 of them:
io.openliberty.beanValidation-3.1_hu.properties
io.openliberty.beanValidation-3.1_pl.properties

**Comment by jhanders34** [comment]
> The feature changes look fine. I'm not qualified to review the NLS properties files in the various languages. I can point out that the english-only words Jakarta Bean Validation aren't fully present in 2 of them: io.openliberty.beanValidation-3.1_hu.properties io.openliberty.beanValidation-3.1_pl.properties

The NLS properties files are a copy of the 3.0 ones with updated copyright and changing 3.0 to 3.1.

File: dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE11Action.java [hunk 2]
```diff
@@ -77,7 +77,7 @@ public class JakartaEE11Action extends FeatureReplacementAction {
                                                   "appAuthorization-3.0",
                                                   "appSecurity-6.0",
                                                   "batch-2.1",
-                                                  "beanValidation-3.0",
+                                                  "beanValidation-3.1",
                                                   "cdi-4.1",
```
Comment (tkburroughs) [inline dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE11Action.java:81]: Should managedBeans-2.0 be removed from this list?

**Review by tkburroughs — CHANGES_REQUESTED** [review]
I think managedBeans-2.0 needs to be removed from server.xml for the Jakarta EE 11 repeat action.

**Comment by tkburroughs** [comment]
Oh, will the removal of managedBeans-2.0, or more specifically, the fact it is no longer compatible with EE 11 cause any problems with the compatibility tests... i.e., the tests that very the different feature combinations?

File: dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE11Action.java (hunk 2 above)
Comment (jhanders34) [reply dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE11Action.java:81]: So removed.

**Comment by jhanders34** [comment]
> Oh, will the removal of managedBeans-2.0, or more specifically, the fact it is no longer compatible with EE 11 cause any problems with the compatibility tests... i.e., the tests that very the different feature combinations?

I haven't done the EE 11 compatibility tests yet.

**Comment by jhanders34** [comment]
#libby

**Comment by jhanders34** [comment]
#libby

**Review by tkburroughs — DISMISSED** [review]

**Review by helyarp — COMMENTED** [review]
Looks good!

**Comment by jhanders34** [comment]
#libby #build

**Review by tkburroughs — APPROVED** [review]
See you found the concurrency test; Nathan was just looking for that during our scrum.

**Comment by posmith** [comment]
L2 message review not required

**Pull request merged**